portfolio/src/main/webapp/recording.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Recording music</title>
+    <link rel="stylesheet" href="styleRecording.css">
```

Comment:
`styleRecording.css` doesn't seem to exist.  You don't necessarily need to include a CSS file per HTML file, so you can simply remove this line.